tests/testthat/test-conjunct_test_linter.R

```diff
@@ -95,3 +95,13 @@ test_that("conjunct_test_linter blocks simple disallowed usages of stopifnot() a
     conjunct_test_linter()
   )
 })
+
+test_that("conjunct_test_linter's allow_named_stopifnot argument works", {
+  # allowed by default
+  expect_lint("stopifnot('x is a logical scalar' = length(x) == 1 && is.logical(x) && !is.na(x))", NULL, conjunct_test_linter())
```

Comment:
nit: IINM the error is taken directly from the argument name, so in production code, the name should be an error message like "`x` must be a logical scalar".